spec.html

```diff
@@ -13799,6 +13892,74 @@ <h1>Runtime Semantics: ArgumentListEvaluation</h1>
       </emu-clause>
     </emu-clause>
 
+    <emu-clause id="sec-optional-chains">
+      <h1>Optional Chains</h1>
+      <emu-note>An optional chain is a chain of one or more property accesses and function calls, the first of which begins with the token `?.`.</emu-note>
+      <emu-clause id="sec-optional-chaining-evaluation">
+        <h1>Runtime Semantics: Evaluation</h1>
+        <emu-grammar>
+          OptionalExpression :
+            MemberExpression OptionalChain
+            CallExpression OptionalChain
+            OptionalExpression OptionalChain
+        </emu-grammar>
+        <emu-alg>
+          1. Let _baseExpression_ be the first child of this production (i.e., this |MemberExpression|, |CallExpression|, or |OptionalExpression|).
+          1. Let _baseReference_ be ? _baseExpression_.Evaluation().
```

Comment:
```suggestion
          1. Let _baseReference_ be the result of evaluating _baseExpression_.
```

(There's no need for the `?`; that's [handled by](https://tc39.es/ecma262/#sec-getvalue) the `GetValue` in subsequent step.)